hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,72 @@
+RELEASE_TYPE: major
+
+Welcome to the next major version of Hypothesis!
+
+There are no new features here, as we release those in minor versions.
+Instead, 4.0 is a chance for us to remove deprecated features (many already
+converted into no-ops), and turn a variety of warnings into errors.
+
+If you were running on the last version of Hypothesis 3.x *without any
+Hypothesis deprecation warnings*, this will be a very boring upgrade.
+**In fact, nothing will change for you at all.**  Per our deprecation
+policy, we have also retained all deprecations added in the last six
+months (since 2018-07-05) without converting them to errors.
+
+
+Removals
+~~~~~~~~
+- ``hypothesis.extra.datetime`` has been removed, replaced by the core
+  date and time strategies.
+- ``hypothesis.extra.fakefactory`` has been removed, replaced by general
+  expansion of Hypothesis' strategies and the third-party ecosystem.
+- The SQLite example database backend has been removed.
+
+Settings
+~~~~~~~~
+- The :obj:`~hypothesis.settings.deadline` is now enforced by default, rather than just
+  emitting a warning when the default (200 millisecond per test case) deadline is exceeded.
```

Comment:
nit?
```suggestion
   emitting a warning when the default (200 milliseconds per test case) deadline is exceeded.
```

Reply:
Not sure why the suggested diffs always add leading space, but I've fixed (and rebased) this now.